lib/storage/src/content_manager/toc/create_collection.rs

```diff
@@ -293,5 +293,7 @@ impl TableOfContent {
                 Err(err) => log::error!("Initialization failed: {}", err),
             }
         });
+        handle.await?;
```

Comment:
Now the collection create API will await for the data copying, isn't it?

Reply:
I don't think this is good decision. With large collection it will block for too long, which is especially critical in distributed env, where you only have a few ticks to complete operation or reelection will be triggered.

The full in-background job was a conscious design decision